contracts/masset/Masset.sol

```diff
@@ -227,7 +203,22 @@ contract Masset is
         uint256 _minOutputQuantity,
         address _recipient
     ) external override nonReentrant whenHealthy returns (uint256 mintOutput) {
-        mintOutput = _mintTo(_input, _inputQuantity, _minOutputQuantity, _recipient);
+        require(_recipient != address(0), "Invalid recipient");
```

Comment:
Are we not using an internal `_mintTo` function to reduce the contract size? The last time we looked at this calling an internal function used less gas which is counter intuitive.

Reply:
I agree with the counter intuitive point which was the rationale for keeping it inline